x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/history/ILMHistoryStore.java

```diff
@@ -59,12 +58,17 @@ public class ILMHistoryStore implements Closeable {
         ).getBytes()
     );
 
-    private final boolean ilmHistoryEnabled;
+    private boolean ilmHistoryEnabled = true;
```

Comment:
Most of the time we make these `volatile` so they are visible immediately across threads

Reply:
Ah, yep, good catch. 71778259957